pkg/feed/tracker.go

```diff
@@ -0,0 +1,27 @@
+package feed
+
+import "github.com/google/oss-rebuild/pkg/rebuild/rebuild"
+
+type Tracker interface {
```

Comment:
i think we need some docs here. the current impl seems to use package-based tracking but i can imagine that we'd want something else in the future.